docs/variants/protectli_vp2420/overview.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+
+## References
+
+## Documentation sections
+
+* [Releases](releases.md) - groups information about all releases.
+* [Building manual](building-manual.md) - describes how to build Dasharo for
+    Protecli 2420.
+* [Initial deployment](initial-deployment.md) - describes initial Dasharo
+    deployment methods (i. e. flashing new firmware) for Protectli VP2420.
+* [Hardware configuration matrix](hardware-matrix.md) - describes the
```

Comment:
The `Recovery` section is missing

Reply:
Added it for both `vp2410` and `vp2420`.